backend/substrapp/tests/query/tests_query_datasample.py

```diff
@@ -486,7 +486,7 @@ def test_bulk_update_data(self):
                                                   description=self.data_description2,
                                                   data_opener=self.data_data_opener2)
 
-        d = DataSample(path=self.data_file)
+        d = DataSample(path=self.data_file_filename)
```

Comment:
Why not using https://github.com/SubstraFoundation/substra-backend/blob/fc247da9557b69f241e6cfae653983e1d09f4fd3/backend/substrapp/tests/query/tests_query_datasample.py#L199:L201